libs/ui-toolkit/arrow-variation/src/lib/arrow-variation.component.html

```diff
@@ -1,6 +1,11 @@
 <div id="emptyDiv" *ngIf="isZero" class="arrow-variation zero" [ngClass]="size"></div>
 <div *ngIf="!isZero" class="arrow" [ngClass]="size">
-  <svg width="100%" height="100%" viewBox="0 0 24 24" style="display: block">
-    <path [attr.fill]="arrowColor" [attr.d]="arrowDirection" />
-  </svg>
+  <ng-container *ngIf="!pdfPrint; else pngArrow">
+    <svg width="100%" height="100%" viewBox="0 0 24 24" style="display: block">
+      <path [attr.fill]="arrowColor" [attr.d]="arrowDirection" />
+    </svg>
+  </ng-container>
+    <ng-template #pngArrow>
+      <img [src]="arrowLink" width="100%" height="100%">
+    </ng-template>
```

Comment:
Check identation, the template should be at the same level of the container that it is related to.